### PR TITLE
feat(electron): add version number to splash screen

### LINKED ISSUE #490: Add version number and potentially other data to splash screen

### PR DESCRIPTION
**Pull request by saihaj**
### Summary of PR
Add version number to splash screen

**Before**
image

**After**
image

### Time spent on PR
40 minutes

### Linked issues
Fixes #490

### Reviewers
@Harjot1Singh @bhajneet

### CODE REVIEW

**Comment by bhajneet** [comment]
Does the ellipsis animation still work?

**Comment by saihaj** [comment]
Yes

**Comment by saihaj** [comment]
Kapture 2020-07-14 at 14 16 22

**Comment by bhajneet** [comment]
UI wise looks fine to me. I will leave code review to @Harjot1Singh

File: app/electron/electron-wrapper.js [hunk 1]
```diff
@@ -14,6 +15,9 @@ app.on( 'ready', () => {
 
   logger.info( 'Loading splashscreen' )
   splashScreen = createSplashScreen()
+  splashScreen.customProps = {
```
Comment (Harjot1Singh) [inline app/electron/electron-wrapper.js:18]: Please rename to `globals`, since it is more descriptive

File: app/electron/splashscreen/index.html [hunk 2]
```diff
@@ -15,8 +15,13 @@
       <span>Shabad OS</span>
     </div>
     <div class="message">
-      <span>Starting</span>
+      <span id="start-message">Starting</span>
     </div>
+    <script>
+      const electron = require('electron');
```
Comment (Harjot1Singh) [inline app/electron/splashscreen/index.html:21]: I know we don't have linting enabled in this environment, but keep to the styleguide here.

`const electron = require( 'electron' )`

Same for the code below.

File: app/electron/splashscreen/index.html [hunk 3]
```diff
@@ -15,8 +15,13 @@
...
+      const { customProps: { version } } = electron.remote.getCurrentWindow();
+      document.getElementById('start-message').innerHTML = `Starting v${version}`
+    </script>
 </body>
 
 </html>
```
Comment (Harjot1Singh) [inline app/electron/splashscreen/index.html:27]: Missing EOF

**Review by Harjot1Singh — APPROVED** [review]

**Pull request merged**